improver_tests/nbhood/circular_kernel/test_CircularNeighbourhood.py

```diff
@@ -88,10 +74,8 @@ def setUp(self):
     def test_basic(self):
         """Test that the plugin returns an iris.cube.Cube."""
 
-        ranges = 2
-        result = CircularNeighbourhood(weighted_mode=False).apply_circular_kernel(
```

Comment:
I've modified these tests so they now call the run method on Neighbourhood plugin, which testes the same functionality as before. The run method takes in the radius in metres rather than the range in grid cells.